linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/oval/fedora.xml

```diff
@@ -0,0 +1,37 @@
+<def-group>
+  <definition class="compliance" id="configure_crypto_policy" version="1">
+    <metadata>
+      <title>Configure System Cryptographic Policies</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>Look for cipher policy correctly configured in /etc/crypto-policy/config.</description>
```

Comment:
I would say `crypto policy`, instead of `cipher policy`, as it defines more than just ciphers.
There are other mentions of `cipher policy` below too.